pkgs/applications/version-management/git-and-tools/conform/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "conform";
+  version = "0.1.0-alpha.25";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "siderolabs";
+    repo = "conform";
+    sha256 = "sha256-WgWgigpqPoEBY4tLjbzK02WFwrCWPGQWJ5eakLv5IWw=";
+  };
+
+  vendorSha256 = "sha256-Oigt7tAK4jhBQtfG1wdLHqi11NWu6uJn5fmuqTmR76E=";
+
+  doCheck = false;
```

Comment:
why?

when I packaged this for my nur this was enough:

`checkInputs = [ git ];`

Reply:
This was my fault. When I first looked I didn't see any tests.